docs/ROADMAP.md

```diff
@@ -12,9 +12,16 @@
 - Correct YAML example on GitHub README
 - Rename in apis.yaml
 
+
 # 7.1.0
 
 - Register JSON Schema for YAML at: https://www.schemastore.org
+- Killer Grafana Dashboard to import
+  -> Screenshot for README.md
+
+# 7.0.2
+
+- Rename WebSocketInterceptor.flow to something else to avoid confusion with flowParser
```

Comment:
why? `webSocketInterceptor.flow` is a `List<WebSocketInterceptorInterface>` and extremely similar to `proxy.flow` which is a `List<Interceptor>`.

`flowParser` will be refactored and renamed anyways by the "Fix `YAMLParsingTest.errorInListItemUniqueness()`" ROADMAP item. After that, there will be a webSocketFlowParser and proxyFlowParser in the JSON schema (with maybe slightly different names).